docs/source/qpu.rst

```diff
@@ -389,9 +394,15 @@ succinct manner:
     # {(0, 5): 0.888, (0, 6): 0.8, (1, 6): 0.837, (1, 7): 0.87, ...}
 
 With these tools provided by the ``Device`` class, users may learn more about Rigetti hardware, and
-construct programs tailored specifically to that hardware. In addition, the ``Device`` class serves
-as a powerful tool for seeding a QVM with characteristics of the device. For more information on
-this, see the next section.
+construct programs tailored specifically to that hardware. The ``Device`` class serves two additional
+uses:
+
+* The ISA associated to Acorn can be modified by the user to target ``CPHASE`` rather than ``CZ`` on
+  any collection of qubit-qubit links not including 1-6.  Passing such a customized ISA to Forest as
+  part of a call to ``.run`` or ``.run_and_measure`` will both enable compilation utilizing ``CPHASE``
+  as a native gate and will disable compilation from ``CPHASE`` to ``CZ``.
```

Comment:
> and will disable compilation from ``CPHASE`` to ``CZ``.

Is this true and/or I don't understand this sentence?

Reply:
I think this is true as written.

In the first part, I meant: you can do what you like, and the compiler will start spitting out `CPHASE` strings equivalent to what you've written.

In the second part, I meant: without making the ISA modification, if you send `CPHASE(2.0)` through the stack, you'll get `RX/2 CZ -RX/2 RZ(2.0) RX/2 CZ -RX/2`. If you do make the modification, `CPHASE(2.0)` stays `CPHASE(2.0)`; or, the conversion from `CPHASE` to `CZ` is disabled.

Reply:
But things still get compiled to CZ where appropriate. It doens't *only* emit cphase

Reply:
here's a second try